tortoise/backends/base/schema_generator.py

```diff
@@ -449,16 +364,104 @@ def _get_table_sql(self, model: "type[Model]", safe: bool = True) -> dict:
                     lines.insert(-1, indent + unique_index_create_sql)
                     m2m_create_string = "\n".join(lines)
             m2m_tables_for_create.append(m2m_create_string)
+        return m2m_tables_for_create
+
+    def _generate_field_default(
+        self, field_object: Field, table_name: str, column_name: str, model: type[Model]
+    ) -> str:
+        auto_now_add = getattr(field_object, "auto_now_add", False)
+        auto_now = getattr(field_object, "auto_now", False)
+        default = field_object.default
+        if default is not None or auto_now or auto_now_add:
+            if not callable(default) and not isinstance(
+                field_object, (UUIDField, TextField, JSONField)
+            ):
+                default = field_object.to_db_value(default, model)
+                try:
+                    return self._column_default_generator(
+                        table_name,
+                        column_name,
+                        self._escape_default_value(default),
+                        auto_now_add,
+                        auto_now,
+                    )
+                except NotImplementedError:
+                    pass
+        return ""
+
+    def _get_table_sql(self, model: type[Model], safe: bool = True) -> dict:
```

Comment:
There are two name patterns in this file:
- _get_*_sql
- _generate_*

What is the difference? Can we just use a single pattern?

Reply:
Use `get_xxx` for sync function, while `generate_*` for async function.